### PR TITLE
Removed unneeded __init__() methods.

### PR DESCRIPTION
**Pull request by sir-sigurd**

### CODE REVIEW

File: django/contrib/auth/base_user.py [hunk 1]
```diff
@@ -52,19 +52,17 @@ class AbstractBaseUser(models.Model):
 
     REQUIRED_FIELDS = []
 
+    # Stores the raw password if set_password() is called so that it can
+    # be passed to password_changed() after the model is saved.
+    _password = None
```
Comment (orf) [inline django/contrib/auth/base_user.py]: Is this safe? Wouldn't `self._password = x` inside a method update the *class* `_password` attribute, and thus all instances of it?

Comment (sir-sigurd) [reply django/contrib/auth/base_user.py]: > Is this safe? 
Yes.

File: django/contrib/gis/db/backends/postgis/operations.py [hunk 2]
```diff
@@ -140,18 +140,13 @@ def select(self):
...
-        prefix = self.geom_func_prefix
-
-        self.collect = prefix + 'Collect'
-        self.extent = prefix + 'Extent'
-        self.extent3d = prefix + '3DExtent'
-        self.length3d = prefix + '3DLength'
-        self.makeline = prefix + 'MakeLine'
-        self.perimeter3d = prefix + '3DPerimeter'
-        self.unionagg = prefix + 'Union'
+    collect = geom_func_prefix + 'Collect'
```
Comment (timgraham) [inline django/contrib/gis/db/backends/postgis/operations.py]: These might be sorted somewhere above `gis_operators` for similarity with the ordering in `spatialite/operations.py`.

File: django/contrib/auth/base_user.py (hunk 1 above)
Comment (timgraham) [reply django/contrib/auth/base_user.py]: If I understand correctly, `AbstractBaseUser._password = x` could update all instances, however, that doesn't seem like code a developer would write.

Comment (orf) [reply django/contrib/auth/base_user.py]: You're quite right, sorry, not sure how I came up with that idea.

**Pull request merged**